### PR TITLE
Fixes bug where save changes modal was shown to user with execute permissions

### PR DESCRIPTION
**Pull request by mabashian**
##### SUMMARY
link https://github.com/ansible/awx/issues/3378

tldr; of these changes are that we now ignore the saved changes modal if the user cannot add/edit the wfjt in question.

User with execute permissions:
execute_user

User with admin permissions:
admin_user

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
 - UI

##### ADDITIONAL INFORMATION
Locally tested the original use case listed in the bug but these changes could impact permissions around adding/editing the visualizer in general so I went ahead and tested the basic use cases (adding/editing WF graph with a superuser/user with admin permissions and viewing the WF graph with a user with the execute role).

### CODE REVIEW

**Review by jakemcdermott — APPROVED** [review]

**Comment by jakemcdermott** [comment]
regate

**Comment by mabashian** [comment]
regate

**Comment by mabashian** [comment]
regate
